src/main/java/io/percy/appium/providers/GenericProvider.java

```diff
@@ -199,7 +200,7 @@ public void getRegionsByXpath(JSONArray elementsArray, List<String> xpaths) {
     public void getRegionsByIds(JSONArray elementsArray, List<String> ids) {
         for (String id : ids) {
             try {
-                MobileElement element = (MobileElement) driver.findElementByAccessibilityId(id);
+                WebElement element = (WebElement) driver.findElement(By.id(id));
```

Comment:
Not using `MobileBy` because it is deprecated in v8 and removed in v9.
Not using `AppiumBy` because it is not available in v7 (current). In next major release we can remove appium v7 support.

Reply:
We need to check usage numbers, we may not be able to remove support fast enough